From 91326a79ff71b9f856585bf8dfdc112527801e68 Mon Sep 17 00:00:00 2001
From: Petko Bordjukov <bordjukov@gmail.com>
Date: Fri, 10 Jun 2016 08:51:08 +0300
Subject: [PATCH 1/3] Build standalone static library

---
 build/standalone.gypi | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/build/standalone.gypi b/build/standalone.gypi
index 6c88409..4f10909 100644
--- a/build/standalone.gypi
+++ b/build/standalone.gypi
@@ -471,6 +471,9 @@
       }],  # fastbuild!=0
     ],
     'target_conditions': [
+      ['_type=="static_library"', {
+        'standalone_static_library': 1,
+      }],
       ['v8_code == 0', {
         'defines!': [
           'DEBUG',
-- 
2.8.3

