From a27a92cbe9db50bceedfffeb43d12345209e4d75 Mon Sep 17 00:00:00 2001
From: Petko Bordjukov <bordjukov@gmail.com>
Date: Fri, 10 Jun 2016 08:57:29 +0300
Subject: [PATCH 2/3] Disable building tests

---
 Makefile      |  6 ++----
 build/all.gyp | 27 ---------------------------
 2 files changed, 2 insertions(+), 31 deletions(-)

diff --git a/Makefile b/Makefile
index a0c08a6..10b2924 100644
--- a/Makefile
+++ b/Makefile
@@ -258,11 +258,9 @@ NACL_ARCHES = nacl_ia32 nacl_x64
 GYPFILES = third_party/icu/icu.gypi third_party/icu/icu.gyp \
 	   build/shim_headers.gypi build/features.gypi build/standalone.gypi \
 	   build/toolchain.gypi build/all.gyp build/mac/asan.gyp \
-	   test/cctest/cctest.gyp test/fuzzer/fuzzer.gyp \
-	   test/unittests/unittests.gyp tools/gyp/v8.gyp \
-	   tools/parser-shell.gyp testing/gmock.gyp testing/gtest.gyp \
+	   tools/gyp/v8.gyp tools/parser-shell.gyp \
 	   buildtools/third_party/libc++abi/libc++abi.gyp \
-	   buildtools/third_party/libc++/libc++.gyp samples/samples.gyp \
+	   buildtools/third_party/libc++/libc++.gyp \
 	   src/third_party/vtune/v8vtune.gyp src/d8.gyp
 
 # If vtunejit=on, the v8vtune.gyp will be appended.
diff --git a/build/all.gyp b/build/all.gyp
index feaf4fe..96820a0 100644
--- a/build/all.gyp
+++ b/build/all.gyp
@@ -8,11 +8,7 @@
       'target_name': 'All',
       'type': 'none',
       'dependencies': [
-        '../samples/samples.gyp:*',
         '../src/d8.gyp:d8',
-        '../test/cctest/cctest.gyp:*',
-        '../test/fuzzer/fuzzer.gyp:*',
-        '../test/unittests/unittests.gyp:*',
       ],
       'conditions': [
         ['component!="shared_library"', {
@@ -20,29 +16,6 @@
             '../tools/parser-shell.gyp:parser-shell',
           ],
         }],
-        ['test_isolation_mode != "noop"', {
-          'dependencies': [
-            '../test/bot_default.gyp:*',
-            '../test/benchmarks/benchmarks.gyp:*',
-            '../test/default.gyp:*',
-            '../test/ignition.gyp:*',
-            '../test/intl/intl.gyp:*',
-            '../test/message/message.gyp:*',
-            '../test/mjsunit/mjsunit.gyp:*',
-            '../test/mozilla/mozilla.gyp:*',
-            '../test/optimize_for_size.gyp:*',
-            '../test/perf.gyp:*',
-            '../test/preparser/preparser.gyp:*',
-            '../test/simdjs/simdjs.gyp:*',
-            '../test/test262/test262.gyp:*',
-            '../test/webkit/webkit.gyp:*',
-            '../tools/check-static-initializers.gyp:*',
-            '../tools/gcmole/run_gcmole.gyp:*',
-            '../tools/jsfunfuzz/jsfunfuzz.gyp:*',
-            '../tools/run-deopt-fuzzer.gyp:*',
-            '../tools/run-valgrind.gyp:*',
-          ],
-        }],
       ]
     }
   ]
-- 
2.8.3

