From 8c92844ee381fc2074204dd52ca3668e3c6fe89f Mon Sep 17 00:00:00 2001
From: Petko Bordjukov <bordjukov@gmail.com>
Date: Fri, 10 Jun 2016 08:58:29 +0300
Subject: [PATCH 3/3] Use the -fPIC flag for the static library

---
 build/standalone.gypi | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/build/standalone.gypi b/build/standalone.gypi
index 4f10909..3a67af3 100644
--- a/build/standalone.gypi
+++ b/build/standalone.gypi
@@ -728,7 +728,7 @@
           [ 'visibility=="hidden" and v8_enable_backtrace==0', {
             'cflags': [ '-fvisibility=hidden' ],
           }],
-          [ 'component=="shared_library"', {
+          [ 'component=="shared_library" or component=="static_library" and (v8_target_arch=="x64" or v8_target_arch=="arm64" or v8_target_arch=="arm")', {
             'cflags': [ '-fPIC', ],
           }],
           [ 'clang==0 and coverage==1', {
-- 
2.8.3

